tests/test_trailedEdgeSources.py

```diff
@@ -0,0 +1,299 @@
+    Given a `TrailedSource`, construct a record of the true values and an
+    Exposure.
+    """
+
+    def __init__(self, bbox, threshold=10.0, exposure=None, **kwds):
+
+        super().__init__(bbox, threshold, exposure, **kwds)
+
+    def addTrailedSource(self, trail, edge=True):
+        """Add a trailed source to the simulation.
+
+         Re-implemented version of
+        `lsst.meas.base.tests.TestDataset.addSource`. Numerically integrates a
+        Gaussian PSF over a line to obtain an image of a trailed source and
+        adds edge flags to the image.
+        """
+        record = self.catalog.addNew()
+        record.set(self.keys["centroid"], trail.center)
+        rng = np.random.default_rng(32)
+        covariance = rng.normal(0, 0.1, 4).reshape(2, 2)
+        covariance[0, 1] = covariance[1, 0]
+        record.set(self.keys["centroid_sigma"], covariance.astype(np.float32))
+        record.set(self.keys["shape"], self.psfShape)
+        record.set(self.keys["isStar"], False)
+
+        # Sum the psf at each
+        numIter = int(2 * trail.length)
+        xp = np.linspace(trail.x0, trail.x1, num=numIter)
+        yp = np.linspace(trail.y0, trail.y1, num=numIter)
+        for (x, y) in zip(xp, yp):
+            pt = Point2D(x, y)
+            im = self.drawGaussian(self.exposure.getBBox(), trail.instFlux,
+                                   lsst.afw.geom.Ellipse(self.psfShape, pt))
+            self.exposure.getMaskedImage().getImage().getArray()[:, :] += im.getArray()
+
+        planes = self.exposure.mask.getMaskPlaneDict()
+        dim = self.exposure.getBBox().getDimensions()
+
+        # Add edge flags to the first and last 20 columns and rows.
+        if edge:
+            for y in range(20):
+                self.exposure.mask.setMaskPlaneValues(planes['EDGE'], 0, dim[0] - 1, y)
+                self.exposure.mask.setMaskPlaneValues(planes['EDGE'], 0, dim[0] - 1, y + dim[1] - 20)
+
+            for y in range(dim[1]):
+                self.exposure.mask.setMaskPlaneValues(planes['EDGE'], 0, 20, y)
+                self.exposure.mask.setMaskPlaneValues(planes['EDGE'], dim[0] - 20, dim[0] - 1, y)
+
+        totFlux = self.exposure.image.array.sum()
+        self.exposure.image.array /= totFlux
+        self.exposure.image.array *= trail.instFlux
+
+        record.set(self.keys["instFlux"], trail.instFlux)
+        self._installFootprint(record, self.exposure.getImage())
+
+        return record, self.exposure.getImage()
+
+
+# Following from test_trailedSources
+@classParameters(length=length, theta=angle, xc=x_coord, yc=y_coord)
+class TrailedEdgeSourcesTestCase(AlgorithmTestCase, lsst.utils.tests.TestCase):
+    """ Test if ext_trailedSources_Naive_flag_edge is set correctly.
+
+    Given a `TrailedSource`, test if the edge flag is set correctly in the
+    source catalog after the
+    'lsst.meas.extensions.trailedSources.Naive.Plugin.makeTrailedSourceMeasurementTask'
```

Comment:
You want backticks here, I believe, not single quotes

Reply:
Fixed